python/ray/experimental/dag/class_node.py

```diff
@@ -79,6 +79,12 @@ def _contains_input_node(self) -> bool:
         return False
 
     def __getattr__(self, method_name: str):
+        # User trying to call .bind() without a bind class method
+        if method_name == "bind" and "bind" not in dir(self._body):
```

Comment:
@ericl addressed, now we don't have the extra `.bind()` on `@serve.deployment`, consistent with Ray Core .bind now.

Only little nit i did here is to detect and surface the same DAGNode base exception message when user tries to `Actor.bind().bind()`.